pkg/router/controller/endpointsubset/converter_test.go

```diff
@@ -0,0 +1,127 @@
+package endpointsubset_test
+
+import (
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	v1 "k8s.io/api/core/v1"
+	discoveryv1beta1 "k8s.io/api/discovery/v1beta1"
```

Comment:
It wouild be helpful if you could use the same import aliases in the test as in the tested code.

Reply:
Done in https://github.com/openshift/router/commit/26cbd10beee116fcf9f8054265a0e72eb213b5e1.